deps/verifier/src/nvidia/mod.rs

```diff
@@ -204,7 +210,12 @@ impl Nvidia {
         let b64_engine = base64::engine::general_purpose::STANDARD;
 
         let cert_chain_vec: Vec<u8> = b64_engine.decode(device.certificate)?;
-        let report_vec: Vec<u8> = hex::decode(device.evidence)?;
+        // Try hex::decode() to see if someone uses the original encoding still.
+        // An Err Result suggests the evidence is base64 encoded so re-try with that.
+        let report_vec: Vec<u8> = match hex::decode(device.evidence.clone()) {
+            Ok(evidence) => evidence,
+            Err(_) => b64_engine.decode(device.evidence)?,
```

Comment:
Do we need to print a `debug!` message here to tell the re-try happens?

Reply:
My first impression is that usual users (not experts) hardly care about what happens here. `warn!` is a high log level and almost always shows. Thus I suggest `debug!`, for those who are interested in this